golem-service-base/src/model.rs

```diff
@@ -2620,6 +2623,41 @@ pub struct InterruptResponse {}
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd, Serialize, Deserialize, Object)]
 pub struct ResumeResponse {}
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd, Serialize, Deserialize, Object)]
+pub struct UpdateResponse {}
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Enum)]
+pub enum UpdateMode {
```

Comment:
as this is in common services namespace i would consider to rename it to WorkerUpdateMode